test/__init__.py

```diff
@@ -28,6 +28,16 @@
 # RFC 3849: 2001:db8::/32 is for documentation only.
 INVALID_SOURCE_ADDRESSES = [("192.0.2.255", 0), ("2001:db8::1", 0)]
 
+# We use timeouts in three different ways in our tests
+#
+# 1. To make sure that the operation timeouts, we can use a short timeout.
+# 2. To make sure that the test does not hang even if the operation should succeed, we
+#    want to use a long timeout, even more so on CI where tests can be really slow
+# 3. To test our timeout logic by using two different values, eg. by using different
+#    values at the pool level and at the request level.
+SHORT_TIMEOUT = 0.001
```

Comment:
Looks like Travis (which is faster than AppVeyor usually) timed out on this `0.001` value. Maybe we need to have two separate values for the `SHORT_TIMEOUT` as well?

Reply:
Oh, `test_ssl_read_timeout` failed, I had not noticed that, sorry. I think the better fix is to switch to `LONG_TIMEOUT` here. Here's my reasoning.

In this test, we:
 1. read the headers -- this should not timeout, so using `LONG_TIMEOUT` makes sense
 2. read the body -- this *should* timeout, so using `SHORT_TIMEOUT` makes sense

However, both timeouts are read timeouts and are controlled by a single value, so we have to compromise. Switching to `LONG_TIMEOUT` is only a compromise for this test, while bumping `SHORT_TIMEOUT` would be a compromise for all tests.

What do you think?

Reply:
+1 LONG_TIMEOUT works for that test.